pkg/chunk/by_key_test.go

```diff
@@ -102,7 +102,7 @@ func TestNWayIntersect(t *testing.T) {
 }
 
 func BenchmarkByKeyLess(b *testing.B) {
-	a := ByKey{dummyChunk(), dummyChunk()}
+	a := ByKey{dummyChunk(model.Now()), dummyChunk(model.Now())}
```

Comment:
even more deterministic to use the same value for these two calls?

Reply:
It would be. I'd like to communicate that the value doesn't matter and that the values can be different.